ast/spec.md

```diff
@@ -149,6 +151,15 @@ interface Identifier <: Expression, Pattern {
 }
 ```
 
+# PrivateName
+
+```js
+interface PrivateName <: Expression, Pattern {
```

Comment:
The grammar of PrivateNames is pretty different from Identifiers, in terms of where they can be used. It's been suggested that the grammar in the specification be refactored to not call it PrivateName but more like PrivateReference, consisting of the two individual tokens. To me, it makes sense that they are treated differently from identifiers.